app/src/main/java/org/systers/mentorship/view/activities/SignUpActivity.kt

```diff
@@ -146,4 +146,9 @@ class SignUpActivity : BaseActivity() {
         startActivity(intent)
         finish()
     }
+
+    override fun onBackPressed() {
```

Comment:
I think you can simply remove `finish()` from line 56 of [`LoginActivity.kt`](https://github.com/anitab-org/mentorship-android/blob/develop/app/src/main/java/org/systers/mentorship/view/activities/LoginActivity.kt) and that will work the same

Reply:
@devansh-299  I have done the required changes. Please review it.